metrics/core/src/main/java/org/hyperledger/besu/metrics/prometheus/MetricsHttpService.java

```diff
@@ -91,11 +91,7 @@ private void validateConfig(final MetricsConfiguration config) {
     router.route("/").method(HttpMethod.GET).handler(this::handleEmptyRequest);
 
     // Endpoint for Prometheus metrics monitoring.
-    router
-        .route("/metrics")
-        .method(HttpMethod.GET)
-        .produces(TextFormat.CONTENT_TYPE_004)
-        .handler(this::metricsRequest);
+    router.route("/metrics").method(HttpMethod.GET).handler(this::metricsRequest);
```

Comment:
Maybe just check for `"text/plain"` instead of completely dropping Prometheus' very specific version?

Reply:
Given we only have one representation to offer we shouldn't be doing any content negotiation (which is what the Accept header is for).  If we had multiple representations, then yes filtering to a particular `Accept` makes sense but if there's a default version, it should just be returned even if it doesn't match `Accept`.

For example this works fine:
```
curl --fail -v -H 'Accept: foo/foo' https://www.google.com/
```
The caller can then decide if they want to process the response based on the `Content-Type` header.